package.json

```diff
@@ -24,7 +24,9 @@
     "angularfire2": "^4.0.0-rc.2",
     "core-js": "^2.4.1",
     "firebase": "^4.3.0",
+    "node-sass": "^4.5.3",
```

Comment:
I think the dependency is needed only on the dev object.

Reply:
Why? Aren't the dependency essential to running the application?